Tests/test_PhyloXML.py

```diff
@@ -718,8 +718,8 @@ def test_events(self):
         evts = self.phyloxml.phylogenies[4].clade.events
         # Container behavior: __len__, __contains__
         self.assertEqual(len(evts), 1)
-        self.assertEqual("speciations" in evts, True)
-        self.assertEqual("duplications" in evts, False)
+        self.assertTrue("speciations" in evts)
+        self.assertFalse("duplications" in evts)
```

Comment:
I may have missed these two last time, but ``assertIn`` and ``assertNotIn`` here please.